src/SDKs/_metadata/eventgrid_data-plane.txt

```diff
@@ -1,11 +1,11 @@
-2018-08-21 21:58:50 UTC
+2018-10-17 13:10:38 UTC
 
 1) azure-rest-api-specs repository information
 GitHub user: Azure
 Branch:      master
-Commit:      c93c4ed1f9eb24ff0b10f63c23ec89310ed89c3f
```

Comment:
@quintinb  Was this code generated from a spec from your branch. If yes, was this file manually modified?

Reply:
The latest change I just pushed has the generated version of this file using generate.cmd pointed at the Azure master branch of azure-rest-api-specs.  The previous version in the change may have been from my fork.

Reply:
While you are at it, could you replace the `generate.cmd` with `generate.ps1` similar to [this](https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/Maps/Management.Maps/generate.ps1) and regen the code

Reply:
Okay, I updated the pull request.  Please double check me on the new generate.ps1.